dbms/src/Storages/Page/stress/workload/MainEntry.cpp

```diff
@@ -0,0 +1,68 @@
+// Copyright 2022 PingCAP, Ltd.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+#include <Storages/Page/stress/workload/PSStressEnv.h>
+#include <Storages/Page/stress/workload/PSWorkload.h>
+
+using namespace DB::PS::tests;
+
+int StressWorkload::mainEntry(int argc, char ** argv)
+{
+    {
+        // in order to trigger REGISTER_WORKLOAD
```

Comment:
Add some comments about why we need trigger `REGISTER_WORKLOAD` here.

Reply:
fixed!